src/rules/tryceratops/mod.rs

```diff
@@ -20,6 +20,7 @@ mod tests {
     #[test_case(Rule::VerboseRaise, Path::new("TRY201.py"); "TRY201")]
     #[test_case(Rule::TryConsiderElse, Path::new("TRY300.py"); "TRY300")]
     #[test_case(Rule::RaiseWithinTry , Path::new("TRY301.py"); "TRY301")]
+    #[test_case(Rule::ErrorInsteadOfException, Path::new("TRY400.py"); "TRY400")]
```

Comment:
Quick tip for the future: in order to get the fixture to run on CI etc., we need to add a test case here via this macro.